usr/share/rear/output/PXE/default/820_copy_to_net.sh

```diff
@@ -16,7 +16,7 @@ case "$scheme" in
         LogPrint "Transferring PXE files to $OUTPUT_URL"
         for result_file in "${RESULT_FILES[@]}" ; do
```

Comment:
@jsmeix is there a reason why a for loop is used here?

We could simplify this like here in 950_copy_result_files.sh:
https://github.com/rear/rear/blob/390fb719c7a2d4008d583245cd0a3e01ce0575b9/usr/share/rear/output/default/950_copy_result_files.sh#L103

Reply:
@vigri 
I canot tell if here a reason why a `for` loop is used.
The `for` loop is there since the very beginning via
https://github.com/rear/rear/commit/d850c4094238a03c9b926b88d7e1582ecd28af52

Reply:
The difference is between `${RESULT_FILES[*]}` and `${RESULT_FILES[@]}`: Only the **`@`** version will correctly handle file names with spaces or other special characters in the file name.  Therefore the `for` loop is safer.

It seems to me that there is room for improvement in handling `lftp`and I suggest to handle that in another PR so that this PR deals only with adding custom parameters to `lftp` without reworking everything else.